ui/src/app/views/workspace-list/component.spec.tsx

```diff
@@ -0,0 +1,73 @@
+import {mount} from 'enzyme';
+import * as React from 'react';
+
+import {WorkspaceList} from './component';
+import {registerApiClient} from 'app/services/swagger-fetch-clients';
+import {WorkspacesApi, ProfileApi} from "generated/fetch/api";
+import {Profile} from 'generated';
+import {ProfileApiStub} from 'testing/stubs/profile-api-stub';
+import {ProfileStubVariables} from 'testing/stubs/profile-service-stub';
+import {navigate, userProfileStore} from 'app/utils/navigation';
+import {
+  WorkspacesApiStub,
+  workspaceStubs,
+  WorkspaceStubVariables
+} from 'testing/stubs/workspaces-api-stub';
+import {waitOneTickAndUpdate} from 'testing/react-test-helpers';
+
+// Mock the navigate function but not userProfileStore
+jest.mock('app/utils/navigation', () => ({
+    ...(jest.requireActual('app/utils/navigation')),
```

Comment:
running jest locally throws an error on this line for me.

Reply:
fixed this!